### ISSUE TITLE
Secure endpoints for SF Mesh Applications

### ISSUE THREAD

**Issue by vyta** [Azure/service-fabric-mesh-preview#274]
Application endpoints on SF Mesh should be easily secured with HTTPS

@ChackDan

**Comment by ChackDan** [comment]
yes, this is in the pipeline. Plan to show this off at the Microsoft  ignite conference (in Sept) and then we would add it to the preview soon after.

**Comment by boppbo** [comment]
Any update on this?
